test/unit/parser_test.rb

```diff
@@ -24,7 +24,8 @@ def test_conjunction
     expression = @parser.parse('Test Eq 10 Not Test Ne 11')
     assert_equal 10.to_s, expression.first[:value]
     assert_equal 11.to_s, expression.last[:value]
-    assert_equal 'Not', expression.last[:conjunction]
+    assert_equal 'And', expression.last[:conjunction]
```

Comment:
@joshcom, this is perhaps the case you'll be most interested in based on your recent work.